lua/lvim/core/autocmds.lua

```diff
@@ -121,6 +121,35 @@ function M.load_defaults()
         end,
       },
     },
+    { -- taken from AstroNvim
```

Comment:
I'm not quite sold on these, at least not the DirOpened one, what happens if you try to open nvim-tree from the dashboard? :thinking:

Reply:
then nvimtree will load by `:NvimTreeToggle` cmd